WORKSPACE

```diff
@@ -91,11 +92,11 @@ load("@rules_cc//cc:repositories.bzl", "rules_cc_dependencies")
 rules_cc_dependencies()
 
 # For `kt_jvm_library`
-load("@io_bazel_rules_kotlin//kotlin:repositories.bzl", "kotlin_repositories")
+load("@rules_kotlin//kotlin:repositories.bzl", "kotlin_repositories")
```

Comment:
New version of `rules_jvm_external` that had to be upgraded for Bazel 7.x needs this repo expressed as `rules_kotlin`. Attempts to preserve the name and map with `repo_mapping = {...}` didn't work at first but it could be investigated further if changing the repo name is intolerable.